paasta_tools/graceful_app_drain.py

```diff
@@ -73,7 +73,8 @@ def main():
         (old_app_live_happy_tasks,
          old_app_live_unhappy_tasks,
          old_app_draining_tasks,
-         ) = get_old_happy_unhappy_draining_tasks(
+         old_app_at_risk_tasks,
```

Comment:
No, it is saying that line 75 is `old_app_draining_tasks` and line 76 is `old_app_at_risk_tasks` (note the `draining` vs `at_risk`). Subtle difference.